src/libraries/Microsoft.PowerFx.LanguageServerProtocol/LanguageServer/LanguageServer.cs

```diff
@@ -73,7 +73,13 @@ public class LanguageServer
 
         private readonly IHostTaskExecutor _hostTaskExecutor;
 
-        [Obsolete("Use the constructor with ILanguageServerOperationHandlerFactory")]
```

Comment:
I think we should leave this in place.  I have updated the Dataverse repo to ignore the warning.